api-server/src/server/utils/get-curriculum.js

```diff
@@ -19,3 +21,29 @@ export function getChallenges() {
       return [...challengeArray, ...flatten(challengesForBlock)];
     }, []);
 }
+
+export function getBlockToChallengeIdMap() {
+  let blockToChallengeIdMap = {};
+
+  Object.values(curriculum).forEach(superBlock => {
+    if (superBlock.blocks) {
+      Object.entries(superBlock.blocks).forEach(([dashedName, block]) => {
+        if (block.challenges) {
+          blockToChallengeIdMap[dashedName] = block.challenges.map(
+            challenge => challenge.id
+          );
+        }
+      });
+    }
+  });
+
+  return blockToChallengeIdMap;
+}
```

Comment:
**Leaving as a comment for post-pr discussion**

This includes the certifications as "blocks". So, Campers could "unclaim" certifications with this endpoint. Seems like an ok "feature", but I am not sure this is what we want.